bundle/manifests/k8s-nim-operator.clusterserviceversion.yaml

```diff
@@ -73,13 +61,18 @@ metadata:
               ]
             },
             "authSecret": "ngc-api-secret",
-            "externalPVC": "",
-            "nimCache": {
-              "name": "meta-llama3-8b-instruct",
-              "profile": ""
+            "storage": {
+              "nimCache": {
+                "name": "meta-llama3-8b-instruct",
+                "profile": ""
+              }
             },
             "scale": {
-              "minReplicas": 1
+              "enabled": false,
+              "hpa": {
+                "minReplicas": 1,
+                "maxReplicas": 10
```

Comment:
This helps me see that I had a misunderstanding.  I thought the `hpa` field would be identical to some common Kubernetes API object.  TY.